### PR TITLE
Add RealMe X (RMX1901) to Official List

### PR DESCRIPTION
**Pull request by raunaqseth**
XDA Thread : https://forum.xda-developers.com/realme-x/development/10-0-resurrection-remix-v8-x-realme-t4156257

Device Source :  https://github.com/raunaqseth/device_realme_RMX1901-1/tree/rr

Kernel Source : https://github.com/raunaqseth/kernel_realme_RMX1901

### CODE REVIEW

File: res/xml/device_maintainers_fragment.xml [hunk 1]
```diff
@@ -1886,6 +1886,13 @@
             app:codename="@string/device_sunfish_codename"
             app:maintainer="@string/device_sunfish_maintainer"
             app:type="phone" />
+	    
+	<com.android.settings.rr.Preferences.DevicePreference
```
Comment (varund7726) [inline res/xml/device_maintainers_fragment.xml:1890]: wrong category. it shouldn't be under google devices

**Review by varund7726 — COMMENTED** [review]
will merge for now and change

File: res/xml/device_maintainers_fragment.xml (hunk 1 above)
Comment (raunaqseth) [reply res/xml/device_maintainers_fragment.xml:1890]: Got it .Thank you very much.

**Comment by varund7726** [comment]
telegram?

**Comment by raunaqseth** [comment]
id: raunaqseth

**Pull request merged**